ydb/core/base/tablet_pipe.h

```diff
@@ -365,6 +378,7 @@ namespace NKikimr {
             bool PreferLocal = false;
             bool CheckAliveness = false;
             bool ExpectShutdown = false;
+            bool SendForLocalOnly = false;
```

Comment:
Config already has `ForceLocal`, in which case pipe will not even try to connect to remote nodes.